libs/ui/search/src/lib/results-list/results-layout.config.ts

```diff
@@ -0,0 +1,39 @@
+import { InjectionToken, Type } from '@angular/core'
+import { ResultsListLayoutEnum } from '@geonetwork-ui/util/shared'
+import { RecordPreviewCardComponent } from '../record-preview-card/record-preview-card.component'
+import { RecordPreviewListComponent } from '../record-preview-list/record-preview-list.component'
+import { RecordPreviewTextComponent } from '../record-preview-text/record-preview-text.component'
+import { RecordPreviewTitleComponent } from '../record-preview-title/record-preview-title.component'
+import { RecordPreviewComponent } from '../record-preview/record-preview.component'
+
+const DEFAULT_ITEM_CLS = 'pb-4'
+const DEFAULT_CONTAINER_CLS = 'gap-4 p-4'
+
+export class ResultsLayoutConfigItem {
+  constructor(
+    public component: Type<RecordPreviewComponent>,
+    public itemClass: string = DEFAULT_ITEM_CLS,
```

Comment:
Having the possibility to specify a css style here would be nice as well, e.g. for having a precise height (this would remove the need for defining custom css classes)